kstreams/custom_types.py

```diff
@@ -1,5 +1,4 @@
-from typing import Any, Callable, Dict, Sequence, Tuple, TypeVar
+from typing import Dict, Sequence, Tuple
```

Comment:
Maybe this file should be called `types`. From the user perspective, importing `custom_types` may be confusing. I would expect to import the types from kstreams. See [starlette types](https://github.com/encode/starlette/blob/master/starlette/types.py)

Reply:
Yes, it is confusing. If the `rename` does not introduce too many changes I will do it